docs/source/user_guide/installation/install-from-source.rst

```diff
@@ -179,32 +309,139 @@ and run the tests to check your installation.
+
+	.. code:: bash
+
+		pip install -e .[all,dev,docs]
+
+	You can also install PyBaMM with "partial rebuilds" enabled. But, due to the ``--no-build-isolation`` flag, you first need to install the build-time dependencies inside the virtual environment:
+
+	.. code:: bash
+
+		pip install scikit-build-core pybind11 casadi cmake
+
+	You can now install PyBaMM in "editable mode" with "partial rebuilds" for development using the following command:
+
+	.. code:: bash
+
+		pip install --no-build-isolation --config-settings=editable.rebuild=true -e .[all,dev,docs]
+
+	If you are using ``zsh`` or ``tcsh``, you would need to use different pattern matching:
+
+	.. code:: bash
+
+		pip install --no-build-isolation --config-settings=editable.rebuild=true -e '.[all,dev,docs]'
+
+
+.. tab:: Windows
+
+	You can install ``virtualenv`` by executing the following command:
+
+	.. code:: bash
+
+		python -m pip install virtualenv
+
+	Create a virtual environment ``venv`` within the PyBaMM root directory:
+
+	.. code:: bash
+
+		python -m virtualenv venv
+
+	You can then “activate” the environment using:
+
+	.. code:: text
+
+		venv\Scripts\activate.bat
+
+	Now, all the calls to pip described below will install PyBaMM and its
+	dependencies into the environment ``venv``. When you are ready to exit
+	the environment and go back to your original system, just type:
+
+	.. code:: bash
+
+		deactivate
+
+	From the ``PyBaMM/`` directory inside the virtual environment, you can install PyBaMM using
+
+	.. code:: bash
+
+		pip install .
+
+	If you intend to contribute to the development of PyBaMM, it is convenient to
+	install in "editable mode", along with all the optional dependencies and useful
+	tools for development and documentation.
+
+	You can install PyBaMM in an "editable mode" using the following command:
+
+	.. code:: bash
+
+		pip install -e .[all,dev,docs]
+
+	You can also install PyBaMM with "partial rebuilds" enabled. But, due to the ``--no-build-isolation`` flag, you first need to install the build-time dependencies inside the virtual environment:
+
+	.. code:: bash
+
+		pip install scikit-build-core pybind11
+
+	You can now install PyBaMM in "editable mode" with "partial rebuilds" for development using the following command:
+
+	.. code:: bash
+
+		pip install --no-build-isolation --config-settings=editable.rebuild=true -e .[all,dev,docs]
+
+.. note::
+
+	The "partial rebuilds" feature is still experimental and may break. To learn more, check out `scikit-build-core's official documentation <https://scikit-build-core.readthedocs.io/en/stable/configuration.html#editable-installs/>`_.
 
 Before you start contributing to PyBaMM, please read the `contributing
 guidelines <https://github.com/pybamm-team/PyBaMM/blob/develop/CONTRIBUTING.md>`__.
 
 Running the tests
 -----------------
 
+.. tab:: GNU/Linux
+
+	Make sure to install ``texlive-latex-extra`` to pass all tests. Otherwise, you can safely ignore the failed tests needing it.
+
```

Comment:
It would be better if we skip the test conditionally based on whether `latex` is installed (run the test only if `latex --version` exits with no errors and returns an exit code of 0). We can remove this line after that.